ocs_ci/deployment/rosa.py

```diff
@@ -75,7 +75,13 @@ def destroy(self, log_level="DEBUG"):
             log_level (str): log level openshift-installer (default: DEBUG)
 
         """
+        # cluster_details = ocm.get_cluster_details(self.cluster_name)
+        # cluster_id = cluster_details.get("id")
```

Comment:
Can we move it under TODO comment?

Reply:
I discussed it with Filip and agreed to merge this PR as it is to unblock managed service deployment. This will be fixed in followup PR.